### PR TITLE
Update json-schema README.md

### PR DESCRIPTION
**Pull request by sschwartz-builder**
Added setup commands for json-schema package.

### CODE REVIEW

File: packages/json-schema/README.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ builder content schema:
 }
 ```
 
+## Setup
```
Comment (mrkoreye) [inline packages/json-schema/README.md:19]:
```suggestion
## Local Setup
```
I think we might want to point out that these steps are not necessary in order to use the schema, but only if you want to generate it locally from.

**Review by mrkoreye — CHANGES_REQUESTED** [review]
Looking at this a little more, I think maybe we should actually add the `dist` file to this repo as well so the schema setup we describe actually works correctly without people having to download and generate it themselves.

**Comment by mrkoreye** [comment]
@steve8708 I am not super familiar with this json schema stuff, was there a reason the `dist` directory with the `json` was removed in this PR: [packages/json-schema/dist/BuilderContent.json](https://github.com/BuilderIO/builder/pull/503/files#diff-23962119e4935dca4eb19a87cd9ff5b2f54179b6bf2fee762f04bcb1fe6d6dcd)?

**Comment by mandx** [comment]
> was there a reason the dist directory with the json was removed

@mrkoreye maybe there was an assumption that running the Makefile will create the `dist/` directory or something like that, but as @sschwartz-builder found out, this is not the case.

**Pull request closed without merging**